packages/@uppy/locales/src/en_US.js

```diff
@@ -164,6 +164,7 @@ en_US.strings = {
   takePicture: 'Take a picture',
   takePictureBtn: 'Take Picture',
   timedOut: 'Upload stalled for %{seconds} seconds, aborting.',
+  traversingSubfolders: 'Traversing %{numSubFolders} subfolder(s) inside "%{folder}"',
```

Comment:
Do we need to inform the user abut this? I think adding a certain folder is a conscious decision that it comes with all files, including the ones in a subfolder. The term "traversing" is also tech jargon and not as user friendly. I would propose the remove it.

Reply:
if a user selects a folder structure that has a lot of files and folders inside of it, then it will take a long time, so i thought it makes sense to show the user that something is happening, or else they might think that it has crashed if it says just "loading" for many minutes. kind of like how all file copy progress dialogs in windows, macos etc show that they are copying something

Reply:
Maybe not, but I think at least we shoulf show something. Maybe we can show the number of files loaded instead?